pyproject.toml

```diff
@@ -8,21 +8,21 @@ documentation = "https://pyasic.readthedocs.io/en/latest/"
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = "^3.8"
+python = "^3.9"
 httpx = ">=0.26.0"
 asyncssh = ">=2.17.0"
 passlib = ">=1.7.4"
 pyaml = ">=23.12.0"
 tomli = { version = ">=2.0.1", python = "<3.11" }
-tomli-w = "1.0.0"
-betterproto = "2.0.0b6"
```

Comment:
Can't use 2.0.0b7, as it breaks parsing with BOS+, as some of their schemas dont conform properly.